static/js/code-highlighter.js

```diff
@@ -417,11 +422,13 @@ var Highlight = new (class Highlight {
     // NOTE: The order here is intentional so that we throw above if the line
     // is not in the document.
     this.selectedLines.add(line);
+    Panel.onSelectedLineChanged(this.selectedLines);
```

Comment:
(In the future we can do something more generic I guess, but this is ok for now)

It'd be great to put this at the end of the method (once we've updated `lastSelectedLine` here and below), to ensure nothing funny happens in between. Not that it should but...